README.md

```diff
@@ -25,7 +25,7 @@ $\frac{5s}{3attempts}$ * $4attempts$ $\approx$ 6.7 seconds for the tree of 10,00
 
 
 The first reason I would think of is that the hardware is different, like my third point above maybe the first search was run on a powerful machine and the second one, while using the same code, was run on a machine with significantly slower internals.
-The average case is something I found online and as such maybe the search function is poorly written and while it does its job it has in reality a much worse asymptotic complexity for trees with amounts of data.
+The average case is something I found online and as such maybe the search function is poorly written and while it does its job it has in reality a much worse asymptotic complexity for larger trees.
```

Comment:
The size of the tree is part of the asymptotic expression, so you can't have different cases for small and large trees.